app/controllers/destinations_controller.rb

```diff
@@ -1,2 +1,16 @@
 class DestinationsController < ApplicationController
+    def index
```

Comment:
The index and create methods for both Destinations and Countries should be done on those specific cards.  Validations should be done once those methods have been merged.